data_measurements/dataset_statistics.py

```diff
@@ -527,12 +489,12 @@ def load_vocab(self):
         # Handling for changes in how the index is saved.
         self.vocab_counts_df = _set_idx_col_names(self.vocab_counts_df)
 
-    def load_or_prepare_text_duplicates(self, save=True, list_duplicates=True):
+    def load_or_prepare_text_duplicates(self, load_only=False, save=True, list_duplicates=True):
```

Comment:
Would it be cleaner to use self.save instead, and change the part where you save this along with the general stats?

Reply:
I've been going back and forth on that. The main issue is that evaluate can be used to output different results. In the case of text duplicates, it can or cannot include all of the duplicates -- we don't want to save the version without duplicates listed (even though we use it in "general stats") because then the cached version won't have the information we need.